datasets/gooaq/gooaq.py

```diff
@@ -43,7 +45,9 @@
 
 _LICENSE = "Licensed under the Apache License, Version 2.0"
 
-_URL = "https://github.com/allenai/gooaq/raw/main/data/qoogle.jsonl"
+_URL = "https://github.com/allenai/gooaq/raw/main/data/gooaq.jsonl"
+
+_SPLITS_URL = "https://github.com/allenai/gooaq/raw/main/data/split.json"
 
 
 class Gooaq(datasets.GeneratorBasedBuilder):
```

Comment:
Could you update the VERSION of the dataset please ? To 1.2.0 for example

You will also need to rename the folder of the dummy data from 1.1.0 to 1.2.0, and replace "1.1.0" by "1.2.0" in the dataset_infos.json